src/api/sirius_api.cpp

```diff
@@ -6035,6 +6054,7 @@ void sirius_linear_solver(void* const* handler__, double const* vkq__, int const
                 100, // iters
                 1e-13 // tol
             );
+            }  // END scope of memory guards
```

Comment:
here the vector of mem. guards can be cleaned

Reply:
done